src/components/ChefsRestaurantCard/ChefsRestaurantCard.module.scss

```diff
@@ -0,0 +1,81 @@
+@import "../../styles/mediaQueries.scss";
+
+.cardContainer {
+  display: flex;
+  flex-flow: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  width: 245px;
+  height: 233px;
+
+  @include smallDesktopScreen {
+    width: 231px;
+    height: 357px;
+  }
+  .cardImageContainer {
+    width: 245px;
+    height: 152.34px;
+
+    @include smallDesktopScreen {
+      width: 231px;
+      height: 224px;
+    }
+    img {
+      width: 100%;
+      height: 100%;
+      object-fit: cover;
+    }
+  }
+
+  .restaurantContentLayout {
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    align-items: center;
+    width: 245px;
+    height: 81px;
+    background-color: rgba(249, 244, 234, 1);
+
+    @include smallDesktopScreen {
+      width: 231px;
+      height: 133px;
+    }
+
+    .restaurantContentContainer {
+      display: flex;
+      flex-flow: row;
+      justify-content: flex-start;
+      align-items: flex-start;
+      width: calc(100% - 32px);
+      height: calc(100% - 32px);
+
+      @include smallDesktopScreen {
+        display: flex;
+        flex-flow: column;
+        justify-content: flex-start;
+        align-items: center;
+        width: 100%;
+        height: 100%;
+        padding-top: 20px;
+      }
+
+      .restaurantTitle {
+        display: flex;
+        flex-flow: row;
+        justify-content: flex-start;
+        align-items: flex-start;
+        font-size: 18px;
+        font-weight: 400;
+        line-height: 21px;
+        letter-spacing: 2.6700000762939453px;
+
+        @include smallDesktopScreen {
+          font-family: Helvetica Neue;
```

Comment:
Font families should probably be set and imported from a scheme and not be set directly inside an element.

Reply:
@nitzanpap done